src/js/main.js

```diff
@@ -1,6 +1,48 @@
-import {animatedScrollTo} from './es6-scroll-to.js';
+console.log("ES5")
+var contactUs = {
+  buttons: document.querySelectorAll(".imageText__button"),
+  ctaSection: document.querySelectorAll('.contactUs'),
+  opened: "contactUs--opened",
+  closed: "contactUs--closed",
 
-document.querySelector('.title').innerHTML = 'Great success!'
-document.querySelector('.text').innerHTML = 'Javascript is working!'
+  init: function () {
+    this.toggleBox();
+  },
 
-animatedScrollTo(500);
\ No newline at end of file
+  slideDown: function (elementToSlideOpen) {
+    elementToSlideOpen.classList.remove(this.closed);
+    elementToSlideOpen.classList.add(this.opened);
+  },
+
+  slideUp: function (elem) {
+    elem.classList.remove(this.opened);
+    elem.classList.add(this.closed);
+  },
+
+  toggleBox: function () {
+    var self = this;
+
+    for (var i = 0; i < self.buttons.length; i++) {
+      try {
```

Comment:
What does this do? Can this whole try catch construct be removed?

Reply:
This changes the scope behavior of i. If we remove the whole try catch construct, self.ctaSection[ii].dispatchEvent(myEvent); won´t work.